itest/godot/ManualFfiTests.gd

```diff
@@ -2,60 +2,39 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-extends Node
+extends TestSuite
 
-func run() -> bool:
-	print("[GD] Test ManualFfi...")
-	var ok = true
-	ok = ok && test_missing_init()
-	ok = ok && test_to_string()
-	ok = ok && test_export()
-
-	print("[GD] ManualFfi tested (passed=", ok, ")")
-	return ok
-
-func test_missing_init() -> bool:
-	return true # TODO: fix dynamic eval
+func test_missing_init():
+	return # TODO: fix dynamic eval
 
 	var expr = Expression.new()
 	var error = expr.parse("WithoutInit.new()")
-	if error != OK:
-		print("Failed to parse dynamic expression")
-		return false
+	if not assert_eq(error, OK, "Failed to parse dynamic expression"):
```

Comment:
please use `!` 🙂

Reply:
Done throughout.

Reply:
By the way, the style guide [disagrees](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_styleguide.html#boolean-operators) with this. The eternal dilemma: consistency with Rust or consistency with Godot?

Reply:
Well, if they have such a strong opinion on it, they should maybe not provide both options 😛
GDScript 2.0 would have been _the_ chance to clean that up.

I've never used `and`, `or` and `not`, and personally don't find them more readable. Since most people working on this codebase are Rust developers, I'd rather use the operator signs.